### PR TITLE
Added Angular Commercial Support column and values

### PR DESCRIPTION
**Pull request by apetroski**
This PR adds the Commercial Support column to the Angular EOL page

### CODE REVIEW

File: products/angular.md [hunk 1]
```diff
@@ -34,6 +36,7 @@ releases:
     releaseDate: 2022-11-16
     lts: 2023-05-03
     latestReleaseDate: 2023-05-03
+    extendedSupport: No
```
Comment (dianjuar) [inline products/angular.md]: This value should be `true` or `false`

```suggestion
    extendedSupport: flase
```

File: products/angular.md [hunk 2]
```diff
@@ -26,6 +27,7 @@ releases:
     eol: 2024-11-08
     latest: "16.2.5"
     latestReleaseDate: 2023-09-13
+    extendedSupport: No
```
Comment (dianjuar) [inline products/angular.md]: This value should be `true` or `false`

```suggestion
    extendedSupport: flase
```

File: products/angular.md [hunk 3]
```diff
@@ -42,6 +45,7 @@ releases:
     releaseDate: 2022-06-02
     lts: 2022-12-02
     latestReleaseDate: 2023-03-13
+    extendedSupport: No
```
Comment (dianjuar) [inline products/angular.md]: This value should be `true` or `false`

```suggestion
    extendedSupport: flase
```

File: products/angular.md [hunk 4]
```diff
@@ -50,6 +54,7 @@ releases:
     latestReleaseDate: 2023-04-06
     releaseDate: 2021-11-03
     lts: 2022-05-04
+    extendedSupport: Yes
```
Comment (dianjuar) [inline products/angular.md]: This value should be `true` or `false`

```suggestion
    extendedSupport: true
```

File: products/angular.md [hunk 5]
```diff
@@ -58,6 +63,7 @@ releases:
     latest: "12.2.17"
     latestReleaseDate: 2022-11-22
     releaseDate: 2021-05-13
+    extendedSupport: Yes
```
Comment (dianjuar) [inline products/angular.md]: This value should be `true` or `false`

```suggestion
    extendedSupport: true
```

File: products/angular.md [hunk 6]
```diff
@@ -66,6 +72,7 @@ releases:
     latest: "11.2.14"
     latestReleaseDate: 2021-05-12
     releaseDate: 2020-11-11
+    extendedSupport: Yes
```
Comment (dianjuar) [inline products/angular.md]: This value should be `true` or `false`

```suggestion
    extendedSupport: true
```

File: products/angular.md [hunk 7]
```diff
@@ -74,6 +81,7 @@ releases:
     latest: "10.2.5"
     latestReleaseDate: 2021-04-21
     releaseDate: 2020-06-24
+    extendedSupport: Yes
```
Comment (dianjuar) [inline products/angular.md]: This value should be `true` or `false`

```suggestion
    extendedSupport: true
```

File: products/angular.md [hunk 8]
```diff
@@ -82,6 +90,7 @@ releases:
     latest: "9.1.13"
     latestReleaseDate: 2020-12-16
     releaseDate: 2020-02-06
+    extendedSupport: Yes
```
Comment (dianjuar) [inline products/angular.md]: This value should be `true` or `false`

```suggestion
    extendedSupport: true
```

File: products/angular.md [hunk 9]
```diff
@@ -97,4 +106,6 @@ months, with 1-3 minor releases for every major release. There is an
 [update guide](https://angular.io/guide/updating "Keeping your Angular projects up-to-date")
 available.
 
+Commercial support is available for all deprecated versions of Angular through the [HeroDevs Never-Ending Support](https://www.herodevs.com/support/nes-angular) initiative. For more information about commercial support options visit [HeroDevs' Website](https://www.herodevs.com/).
```
Comment (dianjuar) [inline products/angular.md]: Let's add some break lines here

```suggestion
Commercial support is available for all deprecated versions of Angular through the 
[HeroDevs Never-Ending Support](https://www.herodevs.com/support/nes-angular) initiative. 
For more information about commercial support options visit [HeroDevs' Website](https://www.herodevs.com/). 
```

**Review by dianjuar — CHANGES_REQUESTED** [review]
`extendedSupport` option should be either a date or a boolean, according to the [documentation](https://github.com/endoflife-date/endoflife.date/blob/master/CONTRIBUTING.md#new-adding-a-new-product)

```md
    # End of extended/commercial support date (optional if extendedSupportColumn is false, else mandatory).
    # Note that extended/commercial support is different from Long-Term Support. Extended/commercial
    # support must be used only when additional support is available after EOL, usually against payment.
    # Use valid dates, and do not add quotes around dates.
    # Alternatively, set to true|false if the date has not been decided yet.
    extendedSupport: 2020-01-01
```

File: products/angular.md (hunk 1 above)
Comment (apetroski) [reply products/angular.md]: fixed

File: products/angular.md (hunk 2 above)
Comment (apetroski) [reply products/angular.md]: fixed

File: products/angular.md (hunk 3 above)
Comment (apetroski) [reply products/angular.md]: fixed

File: products/angular.md (hunk 4 above)
Comment (apetroski) [reply products/angular.md]: fixed

File: products/angular.md (hunk 5 above)
Comment (apetroski) [reply products/angular.md]: fixed

File: products/angular.md (hunk 6 above)
Comment (apetroski) [reply products/angular.md]: fixed

File: products/angular.md (hunk 7 above)
Comment (apetroski) [reply products/angular.md]: fixed

File: products/angular.md (hunk 8 above)
Comment (apetroski) [reply products/angular.md]: fixed

File: products/angular.md (hunk 9 above)
Comment (apetroski) [reply products/angular.md]: fixed

**Review by dianjuar — APPROVED** [review]

**Pull request merged**